ida-customizations/ida-course/final-exam/index.md

```diff
@@ -7,13 +7,11 @@ tags:
 
 # Final Exam
 
-<!--
-
-The Final Exam will be released on **Thursday, July 6th, 2023**.
+The Final Exam will be released on **Thursday, January 4th, 2024**.
```

Comment:
We will have to adapt this page to the new Final Exam split. Probably we should only include at the end a general description of all Final Exam tasks and make sure to word it as "already released" but you have time to submit all tasks you haven't done by February 1st.